### PR TITLE
#23 Add project repository details

### LINKED ISSUE #23: Add project repository details
* Add README.md file
* Add CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by krzysztofzajaczkowski**
closes #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,84 @@
...
+* Spring Boot
+* Angular
+
+Libraries:
+* Ocelot (.NET)
+* MediatR (.NET)
+* MassTransit (.NET)
+* CSharpFunctionalExtensions (.NET)
+* Machine.Specifications (.NET)
+* xUnit (.NET)
```
Comment (RafKulas) [inline README.md]: Do we want to write down all the libraries that we used? Maybe we can split it on libraries per service or each service would have `README.md` with description of its purpose and technology used?

File: README.md [hunk 2]
```diff
@@ -1 +1,84 @@
...
+* GitHub Actions
+* Internal NuGet packages feed
+
+Components:
+* PostgreSQL
+* RabbitMQ
+* Azurite
+* Azure Blob Storage
+
+Concepts:
```
Comment (RafKulas) [inline README.md]: Aren't those more like `Patterns` instead of `Concepts`?

**Review by RafKulas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (krzysztofzajaczkowski) [reply README.md]: Removed for now.

File: README.md (hunk 2 above)
Comment (krzysztofzajaczkowski) [reply README.md]: Removed for now.

**Review by RafKulas — APPROVED** [review]

**Pull request merged**